nestkernel/recording_backend_mpi.cpp

```diff
@@ -349,15 +351,30 @@ nest::RecordingBackendMPI::get_status( DictionaryDatum& ) const
 }
 
 void
-nest::RecordingBackendMPI::set_status( const DictionaryDatum& )
+nest::RecordingBackendMPI::set_status( const DictionaryDatum& datum )
```

Comment:
This change might trigger a "unused variable" warning, thus:
```suggestion
nest::RecordingBackendMPI::set_status( const DictionaryDatum& )
```